extensions/qute/deployment/pom.xml

```diff
@@ -33,6 +33,7 @@
         <dependency>
             <groupId>io.quarkus</groupId>
             <artifactId>quarkus-junit5-internal</artifactId>
+            <scope>test</scope>
```

Comment:
Looks good but I wonder if `test` shouldn't be the default scope for `quarkus-junit5-internal` declared in the BOM?

Reply:
I probably wouldn't mind but we agreed at the beginning not to add scopes to the BOM, as it is considered a bad practice.

Reply:
Please please don't add scopes to the BOM :). It's very very confusing for someone consuming that dependency

Reply:
@geoand @aloubyansky that's ok, I'm not going to change anything... I can live with that ;-). But in general I find it very convenient if "test" dependencies have the `test` scope. In this particular case, 99% of consumers would have to add `<scope>test</scope>`. Just saying...

Reply:
I completely understand. But if you are not expecting that, it can be super confusing